source/includes/_energy-ratings.md

```diff
@@ -30,10 +31,95 @@ curl https://api.gresb.com/api/v1/energy_ratings
 
 ### Create, Update and Remove Energy Rating
 
-To add an energy rating to an asset, send a request to `/entities/{entity_id}/assets/{asset_id}` using PATCH method. Energy ratings are posted annually as outlined in the data dictionary. Each energy rating has to have an ID and a size specified in order to be valid. Energy ratings can be updated with the corresponding `energy_rating_id` and removed by setting the related fields to `null`.
+Previously, participants could submit only one energy rating per asset. As per the 2024 Standard Changes, asset data entry form will be accomodating the submission of **multiple energy ratings per asset**, facilitating more accurate reporting especially meant for residential units.
```

Comment:
Good, but means we have to update it next year. I wonder if mentioning it in the version table is sufficient, and then we can keep this section generic? 

https://github.com/GRESB/api-docs/blob/master/data/versions.yml

same goes for the below 👇